packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -25,6 +25,7 @@ const Series = memo(
     children = null,
     axisId,
     requiresAxis = true,
+    updatePoints,
```

Comment:
Looks like the default for this is `true`, so I think the props should reflect that

https://api.highcharts.com/class-reference/Highcharts.Series#setData

Reply:
Yeah, I was a bit back and forth regarding this. On one side it makes sense to have the default the  same as Highcharts, but on the other hand I thought passing undefined would be more future proof, as Highcharts could decide to flip the default (though it seems pretty unlikely) and this library should just wrap Highcharts, not implement it's own defaults. I'm fine with it either way, you decide 🙂